hortacloud-deployment/Dockerfile

```diff
@@ -0,0 +1,14 @@
+from node:16
+
+RUN apt-get update 
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install
+RUN rm awscliv2.zip
+
+RUN npm install -g aws-cdk
+
+RUN git clone https://github.com/JaneliaSciComp/hortacloud/
```

Comment:
I think David mentioned he'd prefer to have a single Docker environment that can be used to deploy each update rather than building a new image each time. You can add an extra script that runs when a container starts that pulls the .env from s3 and runs this git clone command. Or you can add those steps as instructions in the readme file.